terminologies/metaDataModelType/relationalMetadataModel.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@id": "https://openminds.ebrains.eu/controlledTerms/metaDataModelType/relationalMetadataModel",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MetaDataModelType",
+  "name": "relational metadata model",
+  "definition": "A graph metadata model defines a set of tabular metadata schemas (including their relations) as architectural base of a relational database for describing the products represented in that database.",
```

Comment:
"A **relational** metadata model ..." (copy-paste-error)

Reply:
yep, thanks for noticing. I've updated the description.